numba/tests/test_boundscheck.py

```diff
@@ -204,6 +210,26 @@ def test_fancy_indexing_boundscheck(self):
         with self.assertRaises(IndexError):
             boundscheck(a)
 
+    def test_fancy_indexing_with_modification_boundscheck(self):
+        a = np.arange(3)
+        b = np.arange(4)
+
+        # Check the numpy behavior to ensure the test is correct.
+        with self.assertRaises(IndexError):
+            # TODO: When we raise the same error message as numpy, test that
```

Comment:
Is there a github issue associated with this? I'm worried this TODO will be lost without a proper github issue.

Reply:
A quick look tells me no, there is no issue, but I would not worry about forgetting about it. Basically Numba now simply outputs: IndexError: index is out of bounds, where as Numpy will also print some details: IndexError: index 100 is out of bounds for axis 0 with size 3.  Bounds checking is optional, and only used during development, because it slows vectorization. Printing more details is a conveniencve function, and if it really block somebody, I'm sure they will raise the issue. Don't worry I say :D